src/simd_radix2_fft.h

```diff
@@ -36,6 +36,24 @@
 namespace quadiron {
 namespace simd {
 
+enum class CtGsCase {
+    SIMPLE,
+    NORMAL,
+    EXTREME,
+};
+
+template <typename T>
+inline CtGsCase get_case(T r, T q)
```

Comment:
This function can be declared `constexpr`.

e.g.:
```
template <typename T>
constexpr CtGsCase get_case(T r, T q)
```

(`constexpr` implies `inline` so no need to keep it).